runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/users/register/table.ts

```diff
@@ -0,0 +1,8 @@
+const tableRegisterTranslations = {
+    'Users.Register.Table.Columns.Email': 'Email',
+    'Users.Register.Table.Columns.CreateDate': 'Create date',
+    'Users.Register.Table.Columns.Role': 'Role',
+    'Users.Register.Table.Error.Rows': 'No requests found'
+};
+
+export default tableRegisterTranslations;
```

Comment:
does your ESLint work? Should throw error "newline required at the end of file"

Reply:
There is actually a newline. 🤔